connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -430,7 +434,7 @@ private void maybeCreateTopic(String topic) {
             log.info("Created topic '{}' using creation group {}", newTopic, topicGroup);
         } else {
             log.warn("Request to create new topic '{}' failed", topic);
-            throw new ConnectException("Task failed to create new topic " + topic + ". Ensure "
+            throw new ConnectException("Task failed to create new topic " + newTopic + ". Ensure "
```

Comment:
should be `newTopic` here for error message.

Reply:
You are referring to the inclusion of topic properties besides the topic name. Yeah that's fine.